awsmp/models.py

```diff
@@ -764,6 +764,51 @@ def _convert_terms_to_dict(self) -> dict[str, Any]:
 
         return yaml_config
 
+    @staticmethod
+    def _mask_unchecked_fields(yaml_config: dict[str, Any]) -> dict[str, Any]:
```

Comment:
love the naming here

Reply:
i'd also vote to make this public since there are callers that are expected to use it. `_driver` is technically private at the moment but there are several things that could be moved to a public API.